### PR TITLE
Made small improvements

### PR DESCRIPTION
**Pull request by ZuluPro**

### CODE REVIEW

File: dbbackup/settings.py [hunk 1]
```diff
@@ -1,18 +1,22 @@
...
+import tempfile
+import socket
 import warnings
 from django.conf import settings
 
 DATABASES = getattr(settings, 'DBBACKUP_DATABASES', list(settings.DATABASES.keys()))
 
 # Fake host
-DBBACKUP_FAKE_HOST = getattr(settings, 'DBBACKUP_FAKE_HOST', 'django-dbbackup')
+HOSTNAME = getattr(settings, 'DBBACKUP_HOSTNAME', socket.gethostname())
```
Comment (ZuluPro) [inline dbbackup/settings.py]: I changed ``DBBACKUP_FAKE_HOST``to ``HOSTNAME`` which is more logical for users.
I also use ``socket.gethostname`` for get default

File: dbbackup/settings.py [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 
 # Directory to use for temporary files
-TMP_DIR = getattr(settings, 'DBBACKUP_TMP_DIR', '/tmp')
+TMP_DIR = getattr(settings, 'DBBACKUP_TMP_DIR', tempfile.gettempdir())
```
Comment (ZuluPro) [inline dbbackup/settings.py]: Change default with Python function for better cross-platform

File: dbbackup/settings.py [hunk 3]
```diff
@@ -1,18 +1,22 @@
...
+TMP_FILE_MAX_SIZE = getattr(settings, 'DBBACKUP_TMP_FILE_MAX_SIZE', 10*1024*1024)
+TMP_FILE_READ_SIZE = getattr(settings, 'DBBACKUP_TMP_FILE_READ_SIZE', 1024*1000)
```
Comment (ZuluPro) [inline dbbackup/settings.py]: Create this settings for customize IO performance.

File: dbbackup/settings.py [hunk 4]
```diff
@@ -68,3 +70,7 @@
     BACKUP_DIRECTORY = STORAGE_OPTIONS['location'] = \
         getattr(settings, 'DBBACKUP_BACKUP_DIRECTORY', os.getcwd())
     warnings.warn("DBBACKUP_BACKUP_DIRECTORY is deprecated, use DBBACKUP_STORAGE_OPTIONS['location']", DeprecationWarning)
+
+if hasattr(settings, 'DBBACKUP_FAKE_HOST'):  # noqa
+    warnings.warn("DBBACKUP_FAKE_HOST is deprecated, use DBBACKUP_HOSTNAME", DeprecationWarning)
+    HOSTNAME = settings.DBBACKUP_FAKE_HOST
```
Comment (ZuluPro) [inline dbbackup/settings.py]: Deprecate ``DBBACKUP_FAKE_HOST``

File: docs/index.rst [hunk 5]
```diff
@@ -23,6 +23,12 @@ Contents:
 .. warning::
    django-dbbackup is currently under heavy refactoring, stay tuned for
    new versions and a final 2.0 release.
+
+Compatibility
+-------------
+
+Django Database Backup supports Pypy, Python 2.7, 3.2 to 3.4 and Django greater than
+1.6.
```
Comment (ZuluPro) [inline docs/index.rst]: Fix issue #69

**Comment by benjaoming** [comment]
A little delayed, but :+1: :)

**Pull request merged**